request-manager/grapher/spec.go

```diff
@@ -91,6 +91,12 @@ func ReadConfig(configFile string) (Config, error) {
 
 	for sequenceName, sequence := range cfg.Sequences {
 		sequence.Name = sequenceName
+		for _, arg := range sequence.Args.Static {
+			if arg.Default == "" {
```

Comment:
Oh wait. What if someone writes `default: ""` — that's legit. Might need to do the same pointer trick as for Parallel to differentiate between a default not being provided, and an empty string. Unfortunately that means some refactoring of whatever else uses the ArgSpec type, to expect a pointer to a string rather than just a string.